### PR TITLE
Feat: Support variable overrides per dbt config instance

### PR DESCRIPTION
**Pull request by izeigerman**

### CODE REVIEW

File: sqlmesh/core/config/root.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class Config(BaseConfig):
         username: Name of a single user who should receive approvals/notification, instead of all users in the `users` list.
         pinned_environments: A list of development environment names that should not be deleted by the janitor task.
         loader: Loader class used for loading project files.
+        loader_kwargs: Key-value arguments to pass to the loader instance.
```
Comment (tobymao) [inline sqlmesh/core/config/root.py:53]: is this dbt only?

Comment (tobymao) [reply sqlmesh/core/config/root.py:53]: should we respect it in sqlmesh?

Comment (izeigerman) [reply sqlmesh/core/config/root.py:53]: yeah, why not? It works for both

Comment (izeigerman) [reply sqlmesh/core/config/root.py:53]: we just don't have anything for sqlmesh right now

**Review by crericha — APPROVED** [review]

**Pull request merged**